app/src/main/kotlin/cz/covid19cz/erouska/ui/sandbox/SandboxFragment.kt

```diff
@@ -1,9 +1,18 @@
 package cz.covid19cz.erouska.ui.sandbox
 
+import android.os.Bundle
+import android.view.View
 import cz.covid19cz.erouska.R
 import cz.covid19cz.erouska.databinding.FragmentSandboxBinding
 import cz.covid19cz.erouska.ui.base.BaseFragment
 
-class SandboxFragment : BaseFragment<FragmentSandboxBinding, SandboxVM>(R.layout.fragment_sandbox, SandboxVM::class) {
+class SandboxFragment :
+    BaseFragment<FragmentSandboxBinding, SandboxVM>(R.layout.fragment_sandbox, SandboxVM::class) {
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        viewModel.downloadKeyExport()
```

Comment:
Přidej TODO že se to má udělat na tlačítko (nebo tam tlačítko rovnou udělej jestli to je easy)